src/components/starlight/EditLink.astro

```diff
@@ -12,10 +12,15 @@ const githubEditUrl =
 
 {
 	editUrl && (
-		<a href={githubEditUrl} class="sl-flex">
-			<Icon name="pencil" size="1.2em" />
-			{labels['page.editLink']}
-		</a>
+		<div class="sl-flex">
```

Comment:
Nitpick: For small screens, this probably should contain a row alignment so that the translated texts do not wrap.

Reply:
Good point — made this container wrap in this case, thanks!